src/rpc/rawtransaction.cpp

```diff
@@ -906,7 +906,7 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
     result_0.pushKV("allowed", test_accept_res);
     if (!test_accept_res) {
         if (state.IsInvalid()) {
-            result_0.pushKV("reject-reason", strprintf("%i: %s", state.GetRejectCode(), state.GetRejectReason()));
+            result_0.pushKV("reject-reason", strprintf("%s", state.GetRejectReason()));
```

Comment:
In commit "[validation] Remove REJECT code from CValidationState" (6f8876e155024181a455fc457ae94a38d0da3626)

I don't think it's necessary, but you could potentially mention this change in release notes, in case someone is relying on the RPC string having a certain format.